rspack.main.config.js

```diff
@@ -245,7 +245,9 @@ const config = {
       "sdk-iframe-embedding-ee-plugins": resolveEnterprisePathOrNoop(
         "/sdk-iframe-embedding-plugins",
       ),
-      "sdk-ee-plugins": resolveEnterprisePathOrNoop("/sdk-plugins"),
+      "sdk-ee-plugins": process.env.MB_EDITION === "ee"
+        ? ENTERPRISE_SRC_PATH + "/sdk-plugins"
+        : SRC_PATH + "/lib/sdk-plugins-noop",
```

Comment:
Just curious, why do we need a separate no-op for sdk-plugins?

Reply:
~I think this is a leftover from previous changes I made. I'll remove it if possible~

It's for types :/ TS gets confused by imports that are obfuscated by dynamic aliases